keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifact/DebianArtifactPublisher.kt

```diff
@@ -0,0 +1,88 @@
+package com.netflix.spinnaker.keel.artifact
+
+import com.netflix.frigga.ami.AppVersion
+import com.netflix.spinnaker.igor.ArtifactService
+import com.netflix.spinnaker.keel.api.DeliveryConfig
+import com.netflix.spinnaker.keel.api.artifacts.ArtifactStatus
+import com.netflix.spinnaker.keel.api.artifacts.BuildMetadata
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+import com.netflix.spinnaker.keel.api.artifacts.GitMetadata
+import com.netflix.spinnaker.keel.api.artifacts.PublishedArtifact
+import com.netflix.spinnaker.keel.api.artifacts.VersioningStrategy
+import com.netflix.spinnaker.keel.api.plugins.ArtifactPublisher
+import com.netflix.spinnaker.keel.api.plugins.SupportedArtifact
+import com.netflix.spinnaker.keel.api.plugins.SupportedVersioningStrategy
+import com.netflix.spinnaker.keel.api.support.EventPublisher
+import com.netflix.spinnaker.keel.artifacts.DebianArtifact
+import com.netflix.spinnaker.keel.artifacts.DebianSemVerVersioningStrategy
+import com.netflix.spinnaker.kork.exceptions.IntegrationException
+import org.springframework.stereotype.Component
+
+/**
+ * Built-in keel implementation of [ArtifactPublisher] that does not itself receive/retrieve artifact information
+ * but is used by keel's `POST /artifacts/events` API to notify the core of new Debian artifacts.
+ */
+@Component
+class DebianArtifactPublisher(
+  override val eventPublisher: EventPublisher,
+  private val artifactService: ArtifactService
+) : ArtifactPublisher<DebianArtifact> {
+  override val supportedArtifact = SupportedArtifact("deb", DebianArtifact::class.java)
+
+  override val supportedVersioningStrategies: List<SupportedVersioningStrategy<*>>
+    get() = listOf(
```

Comment:
this can just be `override val supportedVersioningStrategies = listOf(..)` just like the one above, no need to do the `get()`